intermediate/challenge-1/src/main/resources/application.yml

```diff
@@ -0,0 +1,37 @@
+spring:
+  application:
+    name: bus-app
+    external-context-path: /bus-app
+  profiles:
+    active: dev
+  mail:
+    host: smtp.gmail.com
+    port: 587
+    username: supportmail@gmail.com
+    password: password
+    properties:
+      mail:
+        debug: true
+        smtp:
+          starttls:
+            enabled: true
+            required: true
+server:
+  port: 8080
+  error:
+    whitelabel:
+      enabled: false
+  servlet:
+    context-path: /bus-app
+
+springdoc:
+  api-doc:
+    path: /api-doc
+    enabled: true
+  swagger-ui:
+    enabled: true
+    path: /swagger-ui
+
+bus-app:
+  jwt:
+    key: abcd123
```

Comment:
how would you securely inject this key ?

Reply:
In reality this key will be taken from some keyvault based on the environment it deployed. Usually there will be a workflow in devops for adding the key in keyvault. this process changes based on organisation and it is handled by infrastructure team.